cub/cub/util_type.cuh

```diff
@@ -43,28 +43,6 @@
 #  pragma system_header
 #endif // no system header
 
-#include <cuda/__cccl_config> // _CCCL_CUDACC_VER
-#include <cuda/std/limits>
-
-#if !_NVHPC_CUDA
-#  include <cuda_fp16.h>
-#endif
-
-#if !_NVHPC_CUDA && !defined(CUB_DISABLE_BF16_SUPPORT)
-#  include <cuda_bf16.h>
-// cuda_fp8.h transitively includes cuda_fp16.h, so we have to include the header under !CUB_DISABLE_BF16_SUPPORT
-#  if _CCCL_CUDACC_VER >= 1108000
-// cuda_fp8.h resets default for C4127, so we have to guard the inclusion
-#    if defined(_CCCL_COMPILER_MSVC)
-#      pragma warning(push)
-#    endif
-#    include <cuda_fp8.h>
```

Comment:
critical: the removed code used to include fp8 header. Code below relies on `__CUDA_FP8_TYPES_EXIST__`. The change removed fp8 include along with the fp8 types support. This now doesn't compile:

```cpp
#include <cub/block/block_radix_sort.cuh>
#include <cuda_fp8.h>

__global__ void kernel() {
    __nv_fp8_e4m3 data[3];

    using block_sort_t = cub::BlockRadixSort<__nv_fp8_e4m3, 256, 3>;
    using temp_storage_t = block_sort_t::TempStorage;

    __shared__ temp_storage_t temp_storage;
    block_sort_t(temp_storage).Sort(data);
}
```